src/goTools.ts

```diff
@@ -121,9 +121,14 @@ export function getConfiguredTools(goVersion: GoVersion): Tool[] {
 	// Add the linter that was chosen by the user.
 	maybeAddTool(goConfig['lintTool']);
 
-	// Add the language server for Go versions > 1.10 if user has choosen to do so
+	// Add the language server for Go versions > 1.10 if user has choosen to do so.
+	// Respect the go.alternateTools setting.
 	if (goConfig['useLanguageServer'] && goVersion.gt('1.10')) {
-		maybeAddTool('gopls');
+		if (goConfig['alternateTools']['gopls']) {
+			maybeAddTool(goConfig['alternateTools']['gopls']);
```

Comment:
Are we trying to show the "alternate" tool in the `Go: Install/Update Tools` command?
At present this would only work if the alternate tool was set to `go-langserver` right?

Reply:
That's true. Removed this. I like the idea of completely removing support for go-langserver.